src/NWCClient.ts

```diff
@@ -235,9 +235,10 @@ export class NWCClient {
   lud16: string | undefined;
   walletPubkey: string;
   options: NWCOptions;
-  version: string | undefined;
+  encryptionOrVersion: string | undefined;
```

Comment:
can we just use `encryption` - this will otherwise be hard to remove later. We do not need to know the version actually.